app/models/spree/refund_decorator.rb

```diff
@@ -1,6 +1,31 @@
 module Spree::Retail
   module RefundDecorator
     delegate :pos_order_id, to: :payment
+    delegate :pos_refunded, to: :reimbursement
+
+    def perform!
```

Comment:
Sorry, fresh comment here. Are you overriding an existing `perform!` method here, or is this a new interface method?